pkg/k8s/wait.go

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package k8s
+
+import (
+	buildv1alpha1 "github.com/projectriff/system/pkg/apis/build/v1alpha1"
+	requestv1alpha1 "github.com/projectriff/system/pkg/apis/request/v1alpha1"
+	streamv1alpha1 "github.com/projectriff/system/pkg/apis/stream/v1alpha1"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+func WaitUntilReady(watcher watch.Interface) {
+	for {
+		select {
+		case ev := <-watcher.ResultChan():
+			switch ev.Type {
+			case watch.Added, watch.Modified:
+				// TODO create an interface for this
```

Comment:
Looks like a missed TODO. If not, please could you create an issue and link to it.

Reply:
done